gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -50,6 +51,11 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin {
 
   override def init(sc: SparkContext, pluginContext: PluginContext): util.Map[String, String] = {
     val conf = pluginContext.conf()
+    if (conf.get("spark.test.home", "") != "") {
```

Comment:
equals

Reply:
Not all suites set this variable. Do we need to special config for this feature?

Reply:
Add `spark.gluten.sql.ut.statistic` instead.